packages/gateway/src/constants.js

```diff
@@ -1,3 +1,4 @@
+export const CLOUDFLARE_MAX_OBJECT_SIZE = 512 * Math.pow(1024, 2) // 512MB to bytes
```

Comment:
Maybe add "cache" to the name so we know it relates the cache that cloudflare provides!

```suggestion
export const CF_CACHE_MAX_OBJECT_SIZE = 512 * Math.pow(1024, 2) // 512MB to bytes
```